### ISSUE TITLE
Right hand rule

### ISSUE THREAD

**Issue by wiesener** [kriswiner/MPU9250#31]
Hi kriswiner,

could it be that the mpu9250 does not follow the right hand rule?
Kind regards,

Constantin

**Comment by kriswiner** [comment]
No, the mPU9250 should follow this rul. Do you have a specific example where
you think this is not the case? If you look at the data sheet IIRC the axes
do follow the rule. Most sensors do, but not all.

-----Original Message-----
